apispec/ext/marshmallow/swagger.py

```diff
@@ -453,7 +453,7 @@ class Meta:
     :rtype: dict, a JSON Schema Object
     """
     Meta = getattr(schema, 'Meta', None)
-    if getattr(Meta, 'fields', None) or getattr(Meta, 'additional', None):
+    if (getattr(Meta, 'fields', None) or getattr(Meta, 'additional', None)) and not getattr(Meta, 'model', None):
```

Comment:
@khorolets I think 
```python
if (getattr(Meta, 'fields', None) or getattr(Meta, 'additional', None)) \
        and not getattr(Meta, 'model', None):
```
better

You can check this https://www.python.org/dev/peps/pep-0008/

```python
# No extra indentation.
if (this_is_one_thing and
    that_is_another_thing):
    do_something()

# Add a comment, which will provide some distinction in editors
# supporting syntax highlighting.
if (this_is_one_thing and
    that_is_another_thing):
    # Since both conditions are true, we can frobnicate.
    do_something()

# Add some extra indentation on the conditional continuation line.
if (this_is_one_thing
        and that_is_another_thing):
    do_something()
```